### PR TITLE
Feature/step responce

### PR DESCRIPTION
**Pull request by russellgrim**

### CODE REVIEW

File: Firmware/Hexa/CommandDefinitions.cpp [hunk 1]
```diff
@@ -13,17 +13,17 @@ void ledfunc(){
...
+	/*
+		Runs a step responce for the motor currently in the workspace. 
+		INPUT: one float that represents the PWM input sent to the function
+		OUTPUT: None
+	*/
+	// CLI.read float for input 
+	// read the motor
+	// Dev_LA->stepResponceSetup 
+
+	Dev_LA->stepResponseSetup( CLI.readFloat() );
```
Comment (willz1200) [inline Firmware/Hexa/CommandDefinitions.cpp]: Too many comments for one line of code 😍😍

File: Firmware/Hexa/Controller.h [hunk 2]
```diff
@@ -28,7 +28,7 @@ class Controller: public LinearActuator {
 		void streamPosVel(bool toggle);
 		void update();
 		void position();
-		void stepResponseSetup( float input );
+		void stepResponseSetup( unsigned int Speed );
```
Comment (willz1200) [inline Firmware/Hexa/Controller.h]: This could be changed to unsigned char to save some memory

**Review by willz1200 — CHANGES_REQUESTED** [review]
Review

**Comment by russellgrim** [comment]
made some changes ready for review again

**Pull request merged**